atef/widgets/config/utils.py

```diff
@@ -1284,7 +1345,77 @@ def addChild(self, child: TreeItem) -> None:
         child._parent = self
         child._row = len(self._children)
         self._children.append(child)
-        self._columncount = max(child.columnCount(), self._columncount)
+
+    def removeChild(self, child: TreeItem) -> None:
```

Comment:
Where did the camelcase come in? Mimicking qt api?

Reply:
Yeah, this was done in an effort to mimic QTreeWidgetItem and smooth the transition between `AtefItem` and `TreeItem`, though I ended up mixing styles at some point.  I think at this point we can probably convert everything to snake_case, since this is sufficiently hidden behind the `ConfigTreeModel`